Jenkinsfile

```diff
@@ -0,0 +1,134 @@
+pipeline {
+
+    agent {
+
+        label "pmem-csi"
+    }
+
+    options {
+
+        timeout(time: 2, unit: "HOURS")
+
+    }
+
+    environment {
+
+        PMEM_PATH = "/go/src/github.com/intel/pmem-csi"
+        IMAGE_VERSION = "ci-${env.BUILD_ID}"
+        BUILD_IMAGE = "clearlinux-builder"
+        CLUSTER= "ci-$IMAGE_VERSION"
+        GOVM_YAML="_work/$CLUSTER/deployment.yaml"
+        TEST_BOOTSTRAP_IMAGES="localhost:5000/pmem-csi-driver:$IMAGE_VERSION  localhost:5000/pmem-csi-driver-test:$IMAGE_VERSION"
+
+    }
+
+    stages {
+
+        stage('make test') {
+
+            options {
+
+                timeout(time: 60, unit: "MINUTES")
+                retry(3)
+
+            }
+
+            steps {
+
+                sh 'docker build --target build -t $BUILD_IMAGE .'
+                sh 'docker run --rm \
+                -v `pwd`:$PMEM_PATH $BUILD_IMAGE \
+                bash -c "ldconfig; make test"'
```

Comment:
Is the `ldconfig` workaround still necessary? We now have it in the file:
https://github.com/intel/pmem-CSI/blob/4d781f15560cc512f067bf0684f03c68cee05aaa/Dockerfile#L13

Reply:
Yes, it's necessary. Without it, libndctl isn't found when running tests inside the build container. This is different from the pkg-config error.